__tests__/endpoint.test.js

```diff
@@ -259,4 +259,26 @@ describe("Endpoint Tests", () => {
           .expect(404)
       });
 })
+describe("GET /api/users", () => {
+  test("200: returns all data from users when requesting all data", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        body.data.forEach((users) => {
+          expect(typeof users.username).toBe("string");
+          expect(typeof users.name).toBe("string");
+          expect(typeof users.avatar_url).toBe("string");
+        });
+      });
+  });
+  test("404: get 404 response", () => {
+    return request(app)
+      .get("/api/userz")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe(undefined);
+      });
+  });
```

Comment:
This is a generic 404 invalid path and does not relate to this specific endpoint. This should be in its own describe block and should have a msg if you wish to include this test (which I think is a good idea)